chapters/en/slides/chapter4_03_training-best-practices.md

```diff
@@ -43,23 +43,6 @@ This is also known as the catastrophic forgetting problem.
   `"PERSON"`
```

Comment:
In the "problem 1" section above, there's a case where "website" and "person" aren't formatted with capitals and code tags - perhaps make this consistent?